ExampleUsage/EnumExample.cs

```diff
@@ -55,20 +59,98 @@ public void ExampleOfUsingApply()
 				context.Database.ExecuteSqlCommand(migrationSql);
 			}
 		}
-	}
+
+        [Test]        
+        public void CheckIfDescriptionColumnsExists()
```

Comment:
I agree.

TestConfiguration is an integration test and not an unit test. It might be good thing to separate unit test from integration tests a little more clearly I think.